.idea/markdown-navigator/profiles_settings.xml

```diff
@@ -0,0 +1,3 @@
+<component name="MarkdownNavigator.ProfileManager">
```

Comment:
This file probably doesn't belong. `.idea` should be added to .gitignore.

Reply:
I didn't want to mess with your .gitignore, and accidentally added this file manually when checking in. Sorry! Will remove!